### PR TITLE
Django 2 bump

### PR DESCRIPTION
**Pull request by nicholasserra**

### CODE REVIEW

File: runtests.py [hunk 1]
```diff
@@ -45,22 +44,45 @@
...
             'tos': {
                 'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
             }
         },
-        TOS_CACHE_NAME = 'tos' 
-    )
+        'TOS_CACHE_NAME': 'tos' 
+    }
+
+    if django.VERSION >= (1, 10, 0):
```
Comment (karbassi) [inline runtests.py:61]: Based on the [support version](https://www.djangoproject.com/download/#supported-versions), as of April 2020 only **2.2 LTS and newer** are supported.

File: .travis.yml [hunk 2]
```diff
@@ -1,71 +1,47 @@
...
-      env: DJANGO="Django>=1.6,<1.7"
-    - python: "3.6"
-      env: DJANGO="Django>=1.7,<1.8"
-    - python: "3.6"
-      env: DJANGO="Django>=1.8,<1.9"
-    - python: "3.6"
-      env: DJANGO="Django>=1.9,<1.10"
-    - python: "3.6"
-      env: DJANGO="Django>=1.10,<1.11"
+      env: DJANGO="Django>=1.8,<1.11"
```
Comment (karbassi) [inline .travis.yml:32]: Based on the [support version](https://www.djangoproject.com/download/#supported-versions), as of April 2020 only **2.2 LTS and newer** are supported.

File: .travis.yml [hunk 3]
```diff
@@ -1,71 +1,47 @@
...
 env:
-  - DJANGO="Django>=1.4,<1.5"
-  - DJANGO="Django>=1.5,<1.6"
-  - DJANGO="Django>=1.6,<1.7"
-  - DJANGO="Django>=1.7,<1.8"
   - DJANGO="Django>=1.8,<1.9"
   - DJANGO="Django>=1.9,<1.10"
   - DJANGO="Django>=1.10,<1.11"
   - DJANGO="Django>=1.11,<2.0"
+  - DJANGO="Django>=2.0,<2.1"
```
Comment (karbassi) [inline .travis.yml:16]: Based on the [support version](https://www.djangoproject.com/download/#supported-versions), as of April 2020 only **2.2 LTS and newer** are supported.

File: .travis.yml [hunk 4]
```diff
@@ -1,71 +1,47 @@
...
-    - python: "2.6"
-      env: DJANGO="Django>=1.10,<1.11"
-    - python: "2.6"
-      env: DJANGO="Django>=1.11,<2.0"
-    - python: "2.6"
-      env: DJANGO="https://github.com/django/django/archive/master.tar.gz"
+    - python: "2.7"
+      env: DJANGO="Django>=2.0,<2.1"
+    - python: "2.7"
+      env: DJANGO="Django==2.2.12"
```
Comment (karbassi) [inline .travis.yml:26]: Python 2.x is no longer supported. 

[Source](https://www.python.org/dev/peps/pep-0373/)

File: tos/compat.py [hunk 5]
```diff
@@ -50,7 +51,26 @@ def get_cache(cache_name):
         return get_cache(cache_name)
 
 
+def get_middleware_mixin():
+    if django.VERSION >= (1, 10, 0):
+        return deprecation.MiddlewareMixin
+    else:
```
Comment (karbassi) [inline tos/compat.py:57]: Based on the [support version](https://www.djangoproject.com/download/#supported-versions), as of April 2020 only **2.2 LTS and newer** are supported.

File: tos/compat.py [hunk 6]
```diff
@@ -50,7 +51,26 @@ def get_cache(cache_name):
...
+        return object
+
+
+def get_middleware_settings_key():
+    if django.VERSION >= (1, 10, 0):
+        return 'MIDDLEWARE'
+    else:
```
Comment (karbassi) [inline tos/compat.py:64]: Based on the [support version](https://www.djangoproject.com/download/#supported-versions), as of April 2020 only **2.2 LTS and newer** are supported.

File: tos/compat.py [hunk 7]
```diff
@@ -50,7 +51,26 @@ def get_cache(cache_name):
...
+        return 'MIDDLEWARE_CLASSES'
+
+
 if django.VERSION < (1, 5):
     from django.templatetags.future import url
 else:
```
Comment (karbassi) [inline tos/compat.py:70]: Based on the [support version](https://www.djangoproject.com/download/#supported-versions), as of April 2020 only **2.2 LTS and newer** are supported.

File: tos/compat.py [hunk 8]
```diff
@@ -50,7 +51,26 @@ def get_cache(cache_name):
...
     from django.template.defaulttags import url
+
+if django.VERSION >= (1, 10, 0):
+    from django.urls import reverse
+else:
+    from django.core.urlresolvers import reverse
```
Comment (karbassi) [inline tos/compat.py:76]: Based on the [support version](https://www.djangoproject.com/download/#supported-versions), as of April 2020 only **2.2 LTS and newer** are supported.

**Comment by paddatrapper** [comment]
This works as expected for me. Testing with Django 2.2

**Comment by nicholasserra** [comment]
Thanks @paddatrapper !

Hey @frankwiles, I bumped this package to 0.8.0 for this Django 2 compat. Do you think you'll have any free time soon to cut a release of this? Master branch. Thank you!

**Pull request merged**